arch/arm/core/mpu/Kconfig

```diff
@@ -71,6 +71,7 @@ config CUSTOM_SECTION_MIN_ALIGN_SIZE
 
 config ARM_MPU_PXN
 	bool
+	default y
```

Comment:
~~Should probably be `default y if ARM_MPU` to maintain the original logic in the mps3 case or?~~ nevermind, its within `if ARM_MPU`